model/commitq/githubpr/github_pr_message.go

```diff
@@ -0,0 +1,88 @@
+package githubpr
+
+import (
+	"fmt"
+
+	"github.com/mongodb/grip/level"
+	"github.com/mongodb/grip/message"
+)
+
+type GithubMergeMethod string
+
+// valid Github merge methods
+const (
+	GithubMergeMethodMerge  = GithubMergeMethod("merge")
+	GithubMergeMethodSquash = GithubMergeMethod("squash")
+	GithubMergeMethodRebase = GithubMergeMethod("rebase")
+)
+
+type GithubMergePR struct {
+	Owner       string            `bson:"owner,omitempty" json:"owner,omitempty" yaml:"owner,omitempty"`
+	Repo        string            `bson:"repo,omitempty" json:"repo,omitempty" yaml:"repo,omitempty"`
+	Ref         string            `bson:"ref,omitempty" json:"ref,omitempty" yaml:"ref,omitempty"`
+	PRNum       int               `bson:"PR_num,omitempty" json:"PR_num,omitempty" yaml:"PR_num,omitempty"`
```

Comment:
nit: the convention for bson and json tags is all lowercase

Reply:
Done.

Reply:
Done.